src/main/java/org/codehaus/groovy/classgen/AsmClassGenerator.java

```diff
@@ -305,11 +305,6 @@ private void makeInnerClassEntry(ClassNode cn) {
         }
         int mods = adjustedClassModifiersForInnerClassTable(cn);
 
-
-        if (Modifier.isPrivate(mods)) {
```

Comment:
I'm unsure about removing this block as the commit message suggests it was necessary for android support. Does anyone know the history behind this? @melix